sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/AsyncBenchmark.java

```diff
@@ -130,6 +134,8 @@ protected void init() {
     }
 
     void shutdown() {
+        cosmosAsyncContainer.delete().block();
```

Comment:
In our testing subscriptions we can't create a collection with 1M throughput upfront. The workaround we have is to create a collection with 100K throughput and then scale up. Now with this auto deleting collection it means we have to do this manual process everytime.

we don't want to always delete the collection. We should have a config option called `autoCollectionCreateEnabled` when present we should attempt to 
1. create collection if not exist 
2. delete the collection

when `autoCollectionCreateEnabled=false` we shouldn't attempt to create or delete the collection, and if the collection doesn't exist we should fail.

Reply:
Please check current design , it is more seem less in respect to customer expectation , code will only create resource if it is not present , and only delete newly created resource and not the existing